contents/tutorials/churn-rate.md

```diff
@@ -0,0 +1,87 @@
+
+![Lifecycle chart](../images/tutorials/churn-rate/lifecycle-chart.png)
+
+Once set up, we can click on the dormant part of the chart to get a list of dormant (churned) users. We can see user properties and, more importantly, session recordings for these users. This helps us lower churn in the next section.
+
+You can learn more about lifecycle charts in our [product manual](https://posthog.com/manual/lifecycle). 
+
+## Lowering churn
+
+Now that we calculated and visualized the churn rate in our product, we can figure out ways to lower it. Being an all-in-one product OS, PostHog has tools to help you with this, and they connect to your data and the work you’ve already done.
+
+### Session recordings
+
+First on this list is session recordings. From any of the visualizations, we can get a list of users who churned at specific times. Clicking on the charts we created earlier in this tutorial gives us a list of users. We can check this list for recordings of recent sessions of churned users.
+
+Once we find a session recording from a churned user, we can watch their actions leading up to them churning. We might discover, for example, that the pricing page confuses users or that they can’t set up a key feature. We can use this information to improve these problem areas (check out [experiments](https://posthog.com/manual/experimentation)).
+
+### Cohorts
+
+On top of making improvements to problem areas, we can monitor users at high risk to churn. To do this, we can set up a cohort.
+
+To set up a cohort, go to the cohort page and create filters that match the high churn risk users. In our example, the filters are not completing a key event (uploading a file) in the last 30 days and stopping downloading files in the last 15 days (but having done so before).
+
+![Churn risk cohort](../images/tutorials/churn-rate/churn-risk-cohort.png)
+
+Once saved, this provides us with a dynamic list of “high churn risk” users which we can monitor and use in other areas of PostHog.
+
+### Actions
+
+One way we can use a cohort is by combining it with an event to create an action. For example, we can set up an action to trigger when a user in our high churn risk cohort visits a help page. Using [webhooks](https://posthog.com/docs/integrate/webhooks/message-formatting), we can bring these actions into [Slack](https://posthog.com/docs/integrate/webhooks/slack) or wherever our sales or customer success team spends their time to have them reach out to users when the risk action is triggered.
+
+![Churn risk action](../images/tutorials/churn-rate/churn-risk-action.png)
+
+Now we’ve targeted churn on multiple fronts, both by understanding what users are doing before they churn with session recordings and by monitoring users with cohorts and actions. We’ve begun to set up the tools for lowering churn. With continued improvement and experimentation, you’ll hopefully see improvements in your churn rate and the visualizations you’ve set up.
```

Comment:
```suggestion
Now we’ve targeted churn on multiple fronts, both by understanding what users are doing before they churn with session recordings and by monitoring users with cohorts and actions. We’ve begun to set up the tools for lowering churn. With continued improvement and experimentation, you’ll hopefully see improvements in your churn rate and the visualizations you’ve set up.

### Further reading

- [How to build, analyze and optimize conversion funnels in PostHog](/tutorials/funnels)
- [The most useful B2B SaaS product metrics](/blog/b2b-saas-product-metrics)

<NewsletterTutorial compact/>
```

Reply:
Always good to add some links at the end both for UX and SEO. Also, newsletter signup. :D